### PR TITLE
[TH2-4984] Transport protocol support

### PR DESCRIPTION
**Pull request by isengrims**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,8 @@
+<<<<<<< HEAD
```
Comment (Nikita-Smirnov-Exactpro) [inline README.md:1]: Resolve conflicts

File: build.gradle [hunk 2]
```diff
@@ -1,37 +1,74 @@
...
     apply plugin: 'com.palantir.docker'
+    apply plugin: 'application'
 
     ext {
         resourceDir = "src/main/resources"
         sharedDir = file("${project.rootDir}/shared")
 
         baseSailfishVersion           = '3.3'
-        sailfishVersion               = "3.3.54"
+        sailfishVersion               = "3.3.106"
```
Comment (Nikita-Smirnov-Exactpro) [inline build.gradle:17]:
```suggestion
        sailfishVersion               = "3.3.111"
```
use the last release

**Review by Nikita-Smirnov-Exactpro — CHANGES_REQUESTED** [review]

File: build.gradle (hunk 2 above)
Comment (isengrims) [reply build.gradle]: Done

File: README.md (hunk 1 above)
Comment (isengrims) [reply README.md]: Done

**Review by Nikita-Smirnov-Exactpro — APPROVED** [review]

**Pull request merged**